operator/cmd/mesh/install.go

```diff
@@ -94,9 +94,10 @@ func InstallCmd(logOpts *log.Options) *cobra.Command {
 	iArgs := &installArgs{}
 
 	ic := &cobra.Command{
-		Use:   "install",
-		Short: "Applies an Istio manifest, installing or reconfiguring Istio on a cluster.",
-		Long:  "The install command generates an Istio install manifest and applies it to a cluster.",
+		Use:     "install",
+		Short:   "Applies an Istio manifest, installing or reconfiguring Istio on a cluster.",
+		Long:    "The install command generates an Istio install manifest and applies it to a cluster.",
+		Aliases: []string{"apply"},
```

Comment:
This is a great idea. 
Do you think the release note will be good to add for this alias? so that users will know that they can use `istioctl apply` in the same way as they use `kubectl apply`.

Reply:
we used to have `istioctl manifest apply` which we deprecated and removed in `1.7`

Reply:
Yep, just added release note.